### ISSUE TITLE
Sections use first number as grouping

### ISSUE THREAD

**Issue by rallu** [SC5/sc5-styleguide#447]
I have styleguide with main sections from 1-12 and when I click first section 1. it will display also 11. and 12.

It looks like it is using first number to group items.

Simple example KSS to reproduce: 

```javascript
// Section 1
//
// Styleguide 1.0

// Section 11
//
// Styleguide 11.0
```

**Comment by jpbackman** [comment]
Fixed in https://github.com/SC5/sc5-styleguide/pull/449
To be released with 0.3.5

**Comment by jpbackman** [comment]
Reopening until release.
